TypeCobol/Compiler/CupPreprocessor/CompilerDirectiveBuilder.cs

```diff
@@ -85,7 +85,7 @@ public virtual void EnterCopyCompilerStatement(TypeCobol.Compiler.Scanner.Token
             copy.TextName = GetName(qualifiedTextName.TextName);
             copy.TextNameSymbol = qualifiedTextName.TextName;
 #if EUROINFO_RULES
-            if (copy.TextName != null)
+            if (copy.TextName != null && copy.Type != CompilerDirectiveType.EXEC_SQL_INCLUDE)
```

Comment:
I think  this is not the correct place to add this test, it should be with `this.TypeCobolOptions.HasCpyCopy(declaration.TextName)` call (located line 113).

```csharp
if (copy.Type != CompilerDirectiveType.EXEC_SQL_INCLUDE && declaration != null && this.TypeCobolOptions.HasCpyCopy(declaration.TextName))
{
    // Declaration found and copy name starts with Y => apply the legacy REPLACING semantics to the copy directive
    copy.RemoveFirst01Level = true;
	[...]
}
```

Reply:
What I simply understand is that EXEC_SQL_INCLUDE is not concerned with E-I legacy